packages/sdk-starter-kit/src/types.ts

```diff
@@ -44,6 +44,7 @@ export type SafeConfig = {
   owners: string[]
   threshold: number
   saltNonce?: string
+  deploymentType?: DeploymentType
```

Comment:
This will suggest that when Predicting a Safe config the developer can set the deploymentType which is not true